pkg/plugin/builtin/registry.go

```diff
@@ -20,12 +20,18 @@ import (
 	"github.com/conduitio/conduit/pkg/plugin"
 	builtinv1 "github.com/conduitio/conduit/pkg/plugin/builtin/v1"
 	"github.com/conduitio/conduit/pkg/plugin/sdk"
+	"github.com/conduitio/conduit/pkg/plugins/file"
 	"github.com/conduitio/conduit/pkg/plugins/generator"
+	"github.com/conduitio/conduit/pkg/plugins/s3"
+	s3destination "github.com/conduitio/conduit/pkg/plugins/s3/destination"
+	s3source "github.com/conduitio/conduit/pkg/plugins/s3/source"
 )
 
 var (
 	DefaultDispenserFactories = []DispenserFactory{
+		sdkDispenserFactory(file.Specification, file.NewSource, file.NewDestination),
```

Comment:
That's strange, the code is definitely in `lovro/conduit-plugin`, not sure how the diff is calculated here. I hope this will go away after conflicts are resolved.